### DISCUSSION TITLE
[Component Request] - The Timeline Component

### DISCUSSION THREAD

**Discussion by asinghca** [mondaycom/vibe#1990] [Q&A]
Hi Team,

I am working on a app where I would like to use the timeline component as seen on Monday.com.

Is this component available for us to use? Is there a third party library powering this timeline component?

Thank you for your time and consideration :-)

**Comment by talkor** [comment]
Hey @asinghca! Can you please attach a screenshot of that component? You can check out our [component gallery](https://style.monday.com/?path=/docs/catalog--docs) for all the component that are available in this library.
